waku/wms.md

```diff
@@ -93,6 +105,12 @@ A mailserver client has to trust a mailserver, which means they can send direct
 
 A mailserver has a direct TCP connection, which means they are trusted to send traffic. This means a malicious or malfunctioning mailserver can overwhelm an individual node.
 
+## Changelog
```

Comment:
do we not want to have a changelog table like in waku and then reference to tags?

Reply:
sure, it'd be empty now though because there's no 0.1 release and 0.2 isn't done yet (still WIP)